exercises/custom-set/custom_set_test.py

```diff
@@ -0,0 +1,180 @@
+import unittest
+
+from custom_set import CustomSet
+
+class CustomSetTest(unittest.TestCase):
+    def test_empty(self):
+        empty = CustomSet()
+        self.assertTrue(empty.empty())
+    
+    def test_not_empty(self):
+        not_empty = CustomSet(1)
+        self.assertFalse(not_empty.empty())
+    
+    def test_empty_contains_nothing(self):
+        empty = CustomSet()
+        self.assertFalse(1 in empty)
+    
+    def test_simple_contains(self):
+        simple = CustomSet(1,2,3)
+        self.assertTrue(1 in simple)
+        self.assertTrue(2 in simple)
+        self.assertTrue(3 in simple)
+    
+    def test_simple_not_contains(self):
+        simple = CustomSet(1,2,3)
+        self.assertFalse(4 in simple)
+    
+    def test_empty_subset_of_empty(self):
+        empty = CustomSet()
+        self.assertTrue(empty.subset(CustomSet()))
+    
+    def test_empty_subset_of_not_empty(self):
+        empty = CustomSet()
+        not_empty = CustomSet(1)
+        self.assertTrue(not_empty.subset(empty))
+    
+    def test_non_empty_not_subset_of_empty(self):
+        empty = CustomSet()
+        not_empty = CustomSet(1)
+        self.assertFalse(empty.subset(not_empty))
+    
+    def test_set_is_subset_of_itself(self):
+        test_set = CustomSet(1,2,3)
+        self.assertTrue(test_set.subset(CustomSet(1,2,3)))
+    
+    def test_set_is_subset_of_larger(self):
+        test_set = CustomSet(1,2,3)
+        larger = CustomSet(1,2,3,4)
+        self.assertTrue(larger.subset(test_set))
+    
+    def test_set_is_not_subset_of_larger(self):
+        test_set = CustomSet(1,2,3)
+        larger = CustomSet(1,4,5,6)
+        self.assertFalse(larger.subset(test_set))
+
+    def test_empty_disjoint_with_empty(self):
+        empty = CustomSet()
+        self.assertTrue(empty.disjoint(CustomSet()))
+    
+    def test_empty_disjoint_with_not_empty(self):
+        empty = CustomSet()
+        not_empty = CustomSet(1,2,3)
+        self.assertTrue(empty.disjoint(not_empty))
+    
+    def test_not_empty_disjoint_with_empty(self):
+        empty = CustomSet()
+        not_empty = CustomSet(1,2,3)
+        self.assertTrue(not_empty.disjoint(empty))
+    
+    def test_sets_that_share_element_not_disjoint(self):
+        seq = CustomSet(1,2,3)
+        odds = CustomSet(1,3,5)
+        self.assertFalse(odds.disjoint(seq))
+        self.assertFalse(seq.disjoint(odds))
+    
+    def test_sets_that_share_none_disjoint(self):
+        seq = CustomSet(1,2,3)
+        after = CustomSet(4,5,6)
+
+        self.assertTrue(seq.disjoint(after))
+    
+    def test_set_with_same_elems_equal(self):
+        seq = CustomSet(1,2,3)
```

Comment:
You are not testing two sets with the same elements just that the exact same object is equal.